projectfiles/Xcode/The Battle for Wesnoth.xcodeproj/project.pbxproj

```diff
@@ -6543,7 +6558,11 @@
 				EBB44A70837B84A928CB6424 /* edit_pbl_translation.cpp in Sources */,
 				7BFC4DF5BFF8CF75855BA662 /* prompt.cpp in Sources */,
 				C93048A9AE576B6AD016DFA4 /* tod_new_schedule.cpp in Sources */,
+				F13D4C33BAA4CB9E9AACFCC2 /* spinbox.cpp in Sources */,
+				D1254FCA82471825B83AA786 /* multiline_text.cpp in Sources */,
+				E6CF415F9FD04C35A55FB24D /* scroll_text.cpp in Sources */,
 				62714C2FBE84B66CF14E3722 /* test_sdl.cpp in Sources */,
+
```

Comment:
Probably doesn't matter, but if this looks like a leftover from a manual cleanup of an automatic file.